### DISCUSSION TITLE
Add photos and dinstinguish TEL

### DISCUSSION THREAD

**Discussion by frankzou** [XgHao/WeChat-Contact#7] [Ideas]
It may be attractive if you could also export the photo for each contact.
In the CSV, if there is phone number, it could not be recognised. Could you separate it as an individual column?

**Comment by XgHao** [comment]
1. there is currently no plan for the photo, as the CSV format is only a simple separator😴
2. sry, I can't understand what you mean "if there is phone number",  contact info  depends on  the information displayed in the contact detail list, in my wechat-pc, i can't find phone number, so this script can't export phone number. If it's convenient, you can post your WeChat contact detail list and pay attention to desensitization✌️

**Comment by flyinshsky** [comment]
非常不错，感谢作者的辛勤付出，已经测试成功了，非常好，值得表扬赞美，优秀的作者

**Comment by barlow66** [comment]
感谢作者的用心付出和制作  软件非常好用和方便  祈愿所有众生在最需要的时候都能用上好用的工具

**Reply by barlow66** [reply]
再次感谢作者付出

**Comment by zkehao** [comment]
感谢作者，使用过程中有个问题，就是1000个好友导出来也有1000个左右，但是重复的有500多个，所以应该也漏掉了几百个，这个要怎么解决呢？

**Comment by smilepei-jpg** [comment]
<img width="1869" height="981" alt="image" src="https://github.com/user-attachments/assets/a93ff786-3908-408c-8d32-cab42ade0f4c" />
微信的旧版本可以直接在官网下载，导出317个联系人全部成功，但是其中一个联系人自己占了一行，在他下方一行的第一个单元格里又完整出现了全部的信息，把该单元格所在行删除掉之后就正常了。间隔时间调整为500ms时做到一半被弹出了，调整至1234ms后无问题
